### ISSUE TITLE
Custom messages with accent incorrectly displayed

### ISSUE THREAD

**Issue by SybilBoquet** [yikesinc/yikes-inc-easy-mailchimp-extender#338]
For example, if succes message is "Un mail vous a été envoyé" (a mail has been send).
The message is displayed like this : Un mail vous a u00e9tu00e9 envoyu00e9.

**Comment by EvanHerman** [comment]
This issue has been cleared up in the upcoming 6.0 release.


Thanks,
Evan

**Comment by SybilBoquet** [comment]
Sorry, I'm not anderdstand (i'm french).
I use the version 6.0 (downloaded today).
And I have customized the "Success message" with text containing accents.
Accents are incorrectly displayed in form. For example "é" is displayed "u00e9".

Thanks
Laurent

**Comment by ultradour** [comment]
Hi,

I use Easy forms 6.0.0 and I have an accent issue too : sometimes I have to delete/create interest groups because accents are becoming unknowned codes (Brésil is becoming Br00e9sil or something like that). One day this is correct and the day after I have to create interest groups again.
Any idea why this is happenning ?

**Comment by TheTracyL** [comment]
Hello,

Thank you for reporting this bug, We will look into it ASAP!

-Tracy

**Comment by EvanHerman** [comment]
Hi @ultradour and @SybilBoquet,

I've just pushed out an update which properly parses words with accents. There was an issue with the way apostrophes (') and accents (`) were being stored in the database. I've since gone ahead and updated the necessary code, and tested things out on the front end.

I did test with a few strings and the accented letter é.

Give the latest version a shot, and let me know if you are still experiencing the issue. Things should be wrapped up in the latest stable release.

Latest Release Download Link:
https://cloudup.com/files/iU2qFFvl2li/download


Thanks,
Evan
